Source/Core/ClippingPlane.js

```diff
@@ -0,0 +1,155 @@
+define([
+        './Cartesian3',
+        './Check',
+        './defined',
+        './defineProperties'
+    ], function(
+        Cartesian3,
+        Check,
+        defined,
+        defineProperties) {
+    'use strict';
+
+    /**
+     * A Plane in Hessian Normal form to be used with ClippingPlaneCollection.
+     * Compatible with mathematics functions in Plane.js
+     *
+     * @alias ClippingPlane
+     * @constructor
+     *
+     * @param {Cartesian3} normal The plane's normal (normalized).
+     * @param {Number} distance The shortest distance from the origin to the plane.  The sign of
+     * <code>distance</code> determines which side of the plane the origin
+     * is on.  If <code>distance</code> is positive, the origin is in the half-space
+     * in the direction of the normal; if negative, the origin is in the half-space
+     * opposite to the normal; if zero, the plane passes through the origin.
+     */
+    function ClippingPlane(normal, distance) {
```

Comment:
`normal` and `distance` are required, right?  Shouldn't this throw an exception if they are not defined?

Potentially acceptable but borderline shady if this relies on `UpdateChangedCartesian3`'s constructor function to validation `normal`.